src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/events/ManageExternalAppointmentsService.kt

```diff
@@ -90,26 +90,19 @@ class ManageExternalAppointmentsService(
     }
   }
 
+  /**
+   * Amending of appointments is only supported at rolled out prisons, you cannot amend appointments via prison-api.
+   */
   @Transactional
-  fun amendAppointment(
-    oldAppointment: BookingHistoryAppointment,
-    newAppointment: PrisonAppointment,
-  ) {
-    if (activitiesService.isAppointmentsRolledOutAt(oldAppointment.prisonCode)) {
-      activitiesService.findMatchingAppointments(oldAppointment).forEach { matchingId ->
-        log.info("EXTERNAL APPOINTMENTS: patching $oldAppointment in activities and appointments")
-        activitiesService.patchAppointment(matchingId, newAppointment)
-        log.info("EXTERNAL APPOINTMENTS: patched $matchingId in activities and appointments")
-      }
-    } else {
-      // Prison API has no endpoint available to patch an appointment, so in this case we delete the appointment and recreate it
-      prisonService.findMatchingAppointments(oldAppointment).forEach { matchingId ->
-        log.info("EXTERNAL APPOINTMENTS: deleting $oldAppointment from prison-api")
-        prisonService.cancelAppointment(matchingId)
-        log.info("EXTERNAL APPOINTMENTS: deleted $matchingId from prison-api")
-      }
+  fun amendAppointment(oldAppointment: BookingHistoryAppointment, newAppointment: PrisonAppointment) {
```

Comment:
I think this just doesn't make sense for non-rolled out prisons, so felt it better to not allow this to be used for them.